notes-webapp/src/main/webapp/vue-app/notes/components/NotesOverview.vue

```diff
@@ -161,8 +161,8 @@
                   </template>
                   <span class="caption">{{ $t('notes.label.editPage') }}</span>
                 </v-tooltip>
-                <span>{{ $t('notes.label.no-content.no-redactor.content.last') }}</span>
-                <v-tooltip bottom>
+                <span v-if="!note.draftPage">{{ $t('notes.label.no-content.no-redactor.content.last') }}</span>
+                <v-tooltip bottom v-if="!note.draftPage">
```

Comment:
Add `!!note.draftPage` to a computed variable with a name that explains its usage. The name of the variable can simply be `hasDraft` so that the code can be easily maintenable and well understood.

Reply:
done !